nx.json

```diff
@@ -1,9 +1,23 @@
 {
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "parallel": 6,
+  "release": {
```

Comment:
curious how the eventual flow works (as both github actions and nx are involved). didn't really check yet (will do when you have this ready)

Reply:
Here is an nx doc on how it will work: https://nx.dev/recipes/nx-release/publish-in-ci-cd

Essentially:
`nx release` locally, which changes the `package.json` versions.
It will commit & tag with the new version number
The new version number tag is what triggers the workflow to run, which will build and publish the package.